client/src/components/related/PreviewImage.jsx

```diff
@@ -0,0 +1,5 @@
+import React, { useState, useEffect } from 'react';
+
+export default function PreviewImage({ image }) {
+  <img src={image} alt="Primary Product" />;
```

Comment:
For the alt, do you want this to contain a description of what the image is case it doesn't show up? (like "black boots")

Reply:
Thank you! Will change in future pull request